cuequivariance_torch/tests/primitives/equivariant_tensor_product_test.py

```diff
@@ -127,9 +148,16 @@ def test_precision_cuda_vs_fx(
     if not torch.cuda.is_available():
         pytest.skip("CUDA is not available")
 
+    if shared_op0 and batch_size == 0:
+        pytest.skip("shared_op0 requires batch_size > 0")  # TODO: fix this
```

Comment:
@v0i0 uniform1d is not behaving properly when the first operand has batch 1 but the other ones have batch 0.

We would expect the output to be of batch size 0 if at least one input if of batch size 0.